utils/navMenus.js

```diff
@@ -0,0 +1,90 @@
+import Search from "../configuration/config";
+
+import key from "../credentials";
+
+import { menuMore, icons, menus } from "./menuObjects";
+
+const data = {};
+
+/**
+ * Creates an object for each sub-menu that contains a title property and a subArticles property
+ * that contains an object that will generate a number of subArticles for each sub-menu, this number being 5.
+ *
+ *
+ * @param {int} numberOfSubMenus Takes the length of the array of the sub menus to now how many sub menus are.
+ * @param {array} arrayOfSubMenus An array for all the menus of the navbar
+ * @param {string} query A string that will be used to search the articles for each submenu.
+ * @param {int} current Controls the number of sub-menus that each menu should have(5)
+ * @param {int} next Controls the number of sub-menus that each menu should have(5)
+ * @returns {Array} An array of all the subMenus and Articles that have each sub-menu
+ */
+async function createSubMenu(
+  numberOfSubMenus,
+  arrayOfSubMenus,
+  query,
+  current = 0,
+  next = 5
+) {
+  const subArticlesArray = [];
+
+  for (let index = 0; index < numberOfSubMenus; index += 1) {
+    const searchResults = await query.getResults(arrayOfSubMenus[index]);
+    const subArticle = {
+      title: arrayOfSubMenus[index],
+      subArticles: await Promise.all(
```

Comment:
You can remove the `await Promise.all` because you're not returning any promises here.